app/assets/javascripts/discourse/app/models/bookmark.js

```diff
@@ -39,18 +39,10 @@ const Bookmark = RestModel.extend({
   },
 
   attachedTo() {
-    if (this.siteSettings.use_polymorphic_bookmarks) {
-      return {
-        target: this.bookmarkable_type.toLowerCase(),
-        targetId: this.bookmarkable_id,
-      };
-    }
-
-    // TODO (martin) [POLYBOOK] Not relevant once polymorphic bookmarks are implemented.
-    if (this.for_topic) {
-      return { target: "topic", targetId: this.topic_id };
-    }
-    return { target: "post", targetId: this.post_id };
+    return {
+      target: this.bookmarkable_type.toLowerCase(),
```

Comment:
Would `targetType` be a more appropriate name?

Reply:
Actually I need to keep this as target since the `bookmarks:changed` event has been in place for a while, and this would be a breaking change.